aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/DefaultTagger.java

```diff
@@ -0,0 +1,25 @@
+package ru.fix.aggregating.profiler;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+public class DefaultTagger implements Tagger {
```

Comment:
Probably will not be needed if we decide to abolish DEFAULT_TAG_key/value constants: there will be only NoopTagger and RegexpTagger